testutils/file_utils.go

```diff
@@ -18,6 +20,52 @@ import (
 	"go.viam.com/rdk/utils"
 )
 
+const osDarwin = "darwin"
+
+// BuildViamServer will attempt to build the viam-server (full static if on linux). If successful, this function will
+// return the path to the executable.
+func BuildViamServer(tb testing.TB) string {
+	tb.Helper()
+
+	command := "full-static"
```

Comment:
fyi full-static excludes cgo, i.e. is not the complete RDK

if tests are passing I think this is not a problem

Reply:
haha tests weren't passing so changing back to server-static